src/site/_data/macroOptions.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { getAllComponentNames, getMacroOptions } = require('../../lib')
+
+module.exports = async function getAllMacroOptions() {
```

Comment:
This function a bit slow (~900ms for me). There's a message about it when running `npm run build`. Would be interested in how long it takes for you @adamledwards .

Reply:
~1200ms for me